src/components/Item/VisualizationItem/Item.js

```diff
@@ -77,11 +77,9 @@ export class Item extends Component {
 
         this.contentRef = React.createRef();
 
-        this.memoizedApplyFilters = memoizeOne(this.applyFilters);
+        this.memoizedApplyFilters = memoize(this.applyFilters);
```

Comment:
Using lodash memoize breaks the filtering (see the documentation - only the first value of the memoized function is used as the cache key).  So I think we should stick with memoizeOne in this file.